working/augmentation-libraries/feature-specification.md

```diff
@@ -318,6 +318,19 @@ It is a compile-time error if:
     augmentation can both declare a new declaration and augment it in the same
     file.*
 
+### Type inheritance
+
+When a type is omitted in an augmenting declaration, if the first declaration
+that introduced the member has a type specified in the ommitted types position,
+then that type is inherited.
```

Comment:
> if the first declaration ...  has a type specified

Even if the original declaration didn't have a type specified, *it* would have had one inferred, inherited or defaulted. That's the type that gets used for the omitted type of the augmentetion.

Reply:
Fixes here https://github.com/dart-lang/language/pull/3948